partner_used_dealer/models/res_identification_type.py

```diff
@@ -0,0 +1,10 @@
+# Copyright 2022 Quartile Limited
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from odoo import fields, models
+
+
+class ResIdentificationType(models.Model):
+    _name = "res.identification.type"
```

Comment:
```suggestion
    _name = "identification.type"
```
Can we shorten the name.